### ISSUE TITLE
new app in current directory `.`

### ISSUE THREAD

**Issue by jderusse** [symfony/symfony-installer#162]
Creating a new application with directory `.` raise an error
```
$ symfony new .

 Downloading Symfony...

    4.92 MB/4.92 MB ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓  100%

 Preparing project...



                                                                                          
  [RuntimeException]                                                                      
  Symfony can't be installed because the downloaded package is corrupted                  
  or because the installer doesn't have enough permissions to uncompress and              
  rename the package contents.                                                            
  To solve this issue, check the permissions of the /data/perso/editor/app directory and  
  try executing this command again:                                                       
  symfony new .                                                                           
                                                                                          




                                                                                                
  [RuntimeException]                                                                            
  Could not delete /path-to-pwd/.: rmdir(/path-to-pwd/.): Invalid argument  
                                                                                       
```

**Comment by xabbuh** [comment]
The issue actually is that Distill fails when it tries to remove the current working directory (after the archive is uncompressed in a temporary directory under the new project directory which is the current working directory).

**Comment by javiereguiluz** [comment]
Let's ping @raulfraile, the creator of Distill, to ask him if this is something that we could fix in the installer itself or if it's better to fix it upstream in Distill. Thanks!

**Comment by xabbuh** [comment]
see raulfraile/distill#22

**Comment by raulfraile** [comment]
@javiereguiluz @xabbuh thanks for opening the issue! We can keep the conversation there if you like

**Comment by xabbuh** [comment]
@raulfraile :+1: and thank you for the fast reaction

However, we should still keep this issue open as we should bump the required distill version once this is fixed in the library.

**Comment by raulfraile** [comment]
@xabbuh @javiereguiluz I made some changes at https://github.com/raulfraile/distill/commit/eb8fed8e356923d212c0aef2da02e6272ebe78dd , can you try to use the `fix/current_dir` branch to check if it fixes the issue?

**Comment by javiereguiluz** [comment]
@raulfraile sorry for the delay, but I finally managed to test your commit.

First, I needed to copy+paste the few code lines introduced by this change because I couldn't make Composer to install that commit. If I use this version: `"raulfraile/distill": "dev-master#eb8fed8"` I get this error:

```
[RuntimeException]
Failed to execute git checkout 'eb8fed8e356923d212c0aef2da02e6272ebe78dd' -- &&
git reset --hard 'eb8fed8e356923d212c0aef2da02e6272ebe78dd'
fatal: reference is not a tree: eb8fed8e356923d212c0aef2da02e6272ebe78dd
```

Second, I build the PHAR file with the `box build` command.

Third, I test the new installer with the command `symfony.phar new .` and I got the following error:

```
 Downloading Symfony...

  [InvalidArgumentException]
  Identifier "chooser" is not defined.
```

This doesn't make any sense, so please tell me what I did wrong while testing your commit. Thanks.

**Comment by raulfraile** [comment]
@javiereguiluz no problem, please try with this commit https://github.com/raulfraile/distill/commit/5fa7ce1d837953eb26247a4160586c023ae88023 and let me know

**Comment by javiereguiluz** [comment]
This is what I tested:

1. `"raulfraile/distill": "dev-master#5fa7ce1"`  Works perfect
2. `box build` Works perfect
3. `/path/to/new/symfony.phar new .`  Works perfect

However, even if Symfony downloads perfectly and the command doesn't show any error message, the Symfony application is not correctly installed. The `.` directory is empty after executing the command.

**Comment by raulfraile** [comment]
@javiereguiluz thanks for testing it, I'll try to do it by myself later to try to find out what is happening. I'll get back to you as soon as it gets fixed

**Comment by xabbuh** [comment]
I think this is related to the fact that the current working directory is removed during the installation with the current fix.

@javiereguiluz Do you see your application when you first leave the directory and enter it again?

**Comment by raulfraile** [comment]
@javiereguiluz as @xabbuh pointed out, it's because the current working directory is removed and a new directory with the same name is created:

```
$ mkdir test3
$ cd test3
$ ./symfony.phar new .
...
$ ls 
(empty)
$ ls ../test3
README.md
app
bin
composer.json
composer.lock src
vendor
web
```

Let me try to find a solution for this.

**Comment by raulfraile** [comment]
@javiereguiluz can you try again with https://github.com/raulfraile/distill/commit/30e97ed011bb79dc32843b9f27959387221533a9 ? Now I don't remove the current working directory, but the files and then mirror the directory. Let me know if this work (it does for me) and I'll make some cleanups.

**Comment by javiereguiluz** [comment]
@raulfraile everthink works right now. Thank you very much!

These are the step I followed for the test:

1. `"raulfraile/distill": "dev-master#30e97ed"` and `composer update`
2. `./box build` (it generates `symfony.phar`
3. Make a directory for the application (`mkdir test`)
4. Enter into the `test` directory and execute `../symfony.phar new .`
5. Once the installation is finished, `test` contains the Symfony app and you can run `php app/console server:run` from that directory.

**Comment by colinodell** [comment]
+1 I just ran into this issue myself

**Comment by javiereguiluz** [comment]
We want to fix this limitation, but we're waiting for a new release of the Distill library.

**Comment by raulfraile** [comment]
@javiereguiluz sorry for the delay, I'll try to release a new version in a couple of days

**Comment by raulfraile** [comment]
@javiereguiluz I just released a new version of the Distill library (0.9.9)

**Comment by javiereguiluz** [comment]
@raulfraile thanks a lot! I've just created a new PR to fix this issue: #187
